### PR TITLE
Update serial_communication.md

### PR DESCRIPTION
**Pull request by AnotherButler**
Fix broken image link

### CODE REVIEW

File: docs/getting_started/serial_communication.md [hunk 1]
```diff
@@ -95,7 +95,7 @@ int main() {
 
 <span class="tips">**Note:** This example only works if LED1 is on the Pwm pin of the board you are using, such as the NUCLEO-F401RE. </span>
 
-<span class="images">![](NUCLEO-F401RE.PNG)<span>The pin map of the NUCLEO-F401RE shows LED1 on the Pwm pin.</span></span>
+<span class="images">![](images/NUCLEO-F401RE.PNG)<span>The pin map of the NUCLEO-F401RE shows LED1 on the Pwm pin.</span></span>
```
Comment (iriark01) [inline docs/getting_started/serial_communication.md]: The image isn't actually in the images folder

Comment (AnotherButler) [inline docs/getting_started/serial_communication.md]: @iriark01 I think there's one in the images folder and one outside the images folder. Does that matter?

Comment (iriark01) [inline docs/getting_started/serial_communication.md]: There shouldn't be one outside. But also: the new link doesn't seem to work (the image doesn't render), which makes me suspect there isn't one in the images folder.

Comment (AnotherButler) [inline docs/getting_started/serial_communication.md]: I first put it outside, accidentally, and when that didn't work, I added one inside: 
https://github.com/ARMmbed/Handbook/tree/master/docs/getting_started/images

I'll delete the one outside and try uploading the one inside again.

**Comment by iriark01** [comment]
Try just editing the pic's location

**Pull request closed without merging**